services/consensusalgo/leanhelixconsensus/test/leader_elected_by_sync_does_not_immediately_propose_block_test.go

```diff
@@ -24,44 +21,39 @@ import (
 // This causes it to get stuck on GetTransactionsForOrdering (9 seconds when no traffic)
 // and broadcast large pre prepares that nobody cares about to everybody (network pollution).
 
-func TestService_SlowBlockCreationDoesNotHurtBlockSync(t *testing.T) {
+func TestService_ShyLeader_LeaderElectedByBlockSyncDoesNotProposeABlockOnFirstView(t *testing.T) {
 	test.WithContext(func(ctx context.Context) {
 		h := newLeanHelixServiceHarness(0).start(t, ctx)
 
-		isLeanHelixStuckOnCreatingABlock := false
+		h.expectConsensusContextRequestNewBlockNotCalled()
 
-		h.expectConsensusContextRequestOrderingCommittee(0) // we're index 0 (first time called)
-		h.expectConsensusContextRequestOrderingCommittee(0) // we're index 0 (second time called)
-		// TODO REMOVE LINE h.expectGossipSendLeanHelixMessage()
-		h.consensusContextRespondWithVerySlowBlockCreation(&isLeanHelixStuckOnCreatingABlock)
+		h.expectConsensusContextRequestOrderingCommittee(0, 1) // we're index 0 (first time called)
```

Comment:
you're using `RequestOrderingCommittee` as a way to spy on the current view, right? if so, maybe wrap the function with a semantic function `expectCurrentViewToBe`

Reply:
No, I'm using it to enforce leadership of node "0"

Reply:
then rename something like `onRequestOrderingComitteeReturnNodeIndexAsLeader`

Reply:
this works?
`h.beFirstInCommittee() // we will be leader after sync:`